docs/stories/getting-started/usage-examples/top-app-bar-as-container.stories.js

```diff
@@ -0,0 +1,104 @@
+import '@vonage/vwc-side-drawer';
+import '@vonage/vwc-top-app-bar';
+import '@vonage/vwc-top-app-bar-fixed';
+import '@vonage/vwc-icon';
+import '@vonage/vwc-layout';
+import '@vonage/vwc-list';
+import '@vonage/vwc-list/vwc-list-item';
+import '@vonage/vwc-list/vwc-list-expansion-panel';
+import { html } from 'lit-element';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+	title: 'Getting Started/Usage Examples/Top App Bar',
+};
+
+const style = html`
+	<style>
+		.sb-show-main.sb-main-padded {
+			padding: 0;
+		}
+		p {
+			color: "#C0C0C0";
+			font-weight: "bold";
+		}
+		vwc-side-drawer#side-drawer{
+			height: calc(100vh - 64px);
+			--side-drawer-z-index:3;
+		}
+	</style>
+`;
+
+const loremIpsum = () => html`
+<vwc-layout gutters="md">
+	<div>
+		Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+		standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make
+		a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting,
+		remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing
+		Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions
+		of Lorem Ipsum.
+	</div>
+</vwc-layout>
+`;
+
+const content = () => Array(20).fill().map(loremIpsum);
+
+const listItems = () => html`
+	<vwc-list-item shape="rounded" graphic="icon">
+		<vwc-icon slot="graphic" type="chat-line"></vwc-icon>1st level item
+	</vwc-list-item>
+	
+	<p>SECTION TITLE</p>
+	
+	<vwc-list-expansion-panel open>
+		<vwc-list-item slot="header" shape="rounded" graphic="icon">
+			<vwc-icon slot="graphic" type="chat-line"></vwc-icon>1st level item
+		</vwc-list-item>
+		<vwc-list-expansion-panel open>
+			<vwc-list-item slot="header" shape="rounded">2nd level item</vwc-list-item>
+			<vwc-list-item shape="rounded">3rd level item</vwc-list-item>
+			<vwc-list-item shape="rounded">3rd level item</vwc-list-item>
+		</vwc-list-expansion-panel>
+	</vwc-list-expansion-panel>
+`;
+
+const sideDrawerContent = html`
+	<span slot="top-bar">
+		<vwc-icon type="vonage-mono"></vwc-icon>
+		<vwc-text font-face="body-1-bold"> VONAGE</vwc-text>
+	</span>
+	<vwc-list innerRole="navigation" innerAriaLabel="Primary navigation" itemRoles="link">
+		<vwc-list-item shape="rounded" graphic="icon">
+			<vwc-icon slot="graphic" type="home-line"></vwc-icon>1st level item
+		</vwc-list-item>
+	
+		<p>SECTION TITLE</p>
+		${Array(8).fill().map(listItems)}
+	</vwc-list>`;
+
+const topAppBarFixedContent = html`
+	<vwc-button slot="navigationIcon" icon="vonage-mono" label="VONAGE" type="submit" unelevated="">
+		<button type="submit" style="display: none;"></button>
```

Comment:
I'm not suer this line is needed - it suppose to render automatically, if I get it right, whenever` <vwc-button>` is used